config/config.go

```diff
@@ -169,6 +172,33 @@ func DefaultFnConsensusConfig() *FnConsensusConfig {
 	}
 }
 
+type DPOSConfig struct {
+	BootstrapNodes           []string
+	TotalStakedCacheDuration int64
+}
+
+func DefaultDPOSConfig() *DPOSConfig {
+	return &DPOSConfig{
+		BootstrapNodes: []string{
+			"default:0x0e99fc16e32e568971908f2ce54b967a42663a26",
+			"default:0xac3211caecc45940a6d2ba006ca465a647d8464f",
+			"default:0x69c48768dbac492908161be787b7a5658192df35",
```

Comment:
This doesn't seem to match any candidate, where did you get the address from?

Reply:
https://github.com/loomnetwork/dashboard/blob/b2ceea2b21f7620ca6a0b19e38ae8cd05d521f58/src/config/production.ts#L25